### PR TITLE
RIFEX-86 Refactor Pay to RNS Page

### PR DESCRIPTION
**Pull request by JONAF2103**
This PR introduces the design refactor and finishes the pay with lumino feature. We also introduce mocks mode for designers to have an easy way to test without having lumino nodes running locally.

Here we have some screenshots, the idea is to have the functional skeleton so the styles are made by the designer later:

imagen

imagen

imagen

This affects only the ui

### CODE REVIEW

File: ui/app/rif/actions/index.js [hunk 1]
```diff
@@ -678,27 +683,35 @@ function createPayment (partner, tokenAddress, netAmount, callbackHandlers = new
...
   return (dispatch) => {
     return new Promise((resolve, reject) => {
-      background.rif.lumino.getChannels((error, channels) => {
-        if (error) {
-          dispatch(niftyActions.displayWarning(error));
-          return reject(error);
-        }
-        return resolve(channels);
-      });
+      if (rifConfig.mocksEnabled) {
```
Comment (marcosmartinez7) [inline ui/app/rif/actions/index.js:686]: After the design refactor, this code will be deleted right? Maybe add a TODO here?

File: ui/app/rif/actions/index.js [hunk 2]
```diff
@@ -678,27 +683,35 @@ function createPayment (partner, tokenAddress, netAmount, callbackHandlers = new
...
   return (dispatch) => {
     return new Promise((resolve, reject) => {
-      background.rif.lumino.getTokens((error, tokens) => {
-        if (error) {
-          dispatch(niftyActions.displayWarning(error));
-          return reject(error);
-        }
-        return resolve(tokens);
-      });
+      if (rifConfig.mocksEnabled) {
```
Comment (marcosmartinez7) [inline ui/app/rif/actions/index.js:704]: Same here

**Review by marcosmartinez7 — APPROVED** [review]
lgtm

**Review by rorolopetegui — APPROVED** [review]
lgtm

**Pull request merged**